src/bitcoind/mod.rs

```diff
@@ -28,6 +28,9 @@ use jsonrpc::{
 /// A retry happens every 1 second, this makes us give up after one minute.
 const BITCOIND_RETRY_LIMIT: usize = 60;
 
+/// The minimum bitcoind version that can be used with revaultd.
+const MIN_BITCOIND_VERSION: u64 = 210000;
```

Comment:
We are using some features from 22.0, so can you make it `220000`?

Reply:
Ah sure, we will need to also change [this](https://github.com/revault/revaultd/blob/master/README.md#:~:text=(version%200.21%20minimum)) then.

Reply:
Oh, yes, good catch!